service/keypairmanager.go

```diff
@@ -63,6 +69,16 @@ func (db *DB) CreateKeypairTable() error {
 	return err
 }
 
+// AlterKeypairTable adds extra fields to an existing keypair database table
+func (db *DB) AlterKeypairTable() error {
+	_, err := db.Exec(alterKeypairAddAssertion)
+	if err != nil {
+		// Ignore error as the field may already be added
+		return nil
+	}
+	return err
```

Comment:
??? weird method. One improvement could be warning about the error. But in any case, you are always returning nil, no matter whether there is an error or not. Why not:
func (db *DB) AlterKeypairTable() {
   db.Exec(alterKeypairAddAssertion)
}